iothub_client/devdoc/requirement_docs/iothubclient_requirements.md

```diff
@@ -378,12 +378,14 @@ Options handled by IoTHubClient_SetOption:
 
 **SRS_IOTHUBCLIENT_41_003: [** The value of `OPTION_DO_WORK_FREQUENCY_IN_MS` shall be limited to 100 to follow SDK best practices by not reducing the DoWork frequency below 10 Hz **]** 
 
-**SRS_IOTHUBCLIENT_41_004: [** If `currentMessageTimeout` is not less than `do_work_freq_ms`, `IotHubClientCore_SetOption` shall return `IOTHUB_CLIENT_ERROR` **]**
+**SRS_IOTHUBCLIENT_41_004: [** If `currentMessageTimeout` is less than `do_work_freq_ms`, `IotHubClientCore_SetOption` shall return `IOTHUB_CLIENT_ERROR` **]**
```

Comment:
1. Isn't [ `currentMessageTimeout` is not greater than `do_work_freq_ms`], the correct usage?
2. Isn't IOTHUB_CLIENT_INVALID_ARG, the more appropriate return value?

Reply:
I agree. Updating comments to reflect this.